base/loading.jl

```diff
@@ -316,16 +316,20 @@ function pathof(m::Module)
 end
 
 """
-    pkgdir(m::Module)
+    pkgdir(m::Module[, paths::String...])
 
- Return the root directory of the package that imported module `m`,
- or `nothing` if `m` was not imported from a package.
- """
-function pkgdir(m::Module)
+Return the root directory of the package that imported module `m`,
+or `nothing` if `m` was not imported from a package. The optional
+argument `paths` can be used to access subdirs of the module root.
```

Comment:
```suggestion
Return the root directory of the package that imported module `m`,
or `nothing` if `m` was not imported from a package. Optionally further
path component strings can be provided to construct a path within the 
package root.

```julia
julia> pkgdir(Foo)
"/path/to/Foo.jl"

julia> pkgdir(Foo, "src", "file.jl")
"/path/to/Foo.jl/src/file.jl"
```